From ebe149bde14f9299402a5af0024a165c25fe4352 Mon Sep 17 00:00:00 2001
From: Yuval Caduri <cyuval@marvell.com>
Date: Tue, 1 May 2018 23:57:16 +0300
Subject: [PATCH 091/386] dts: marvell: musdk: add XOR UIO nodes

XOR used by user space networking applications.
Added 2 XOR UIO's for each CP and 4 UIO's for AP.

Change-Id: Ib2f5aadf42d56d6cf32d3ab94a93d552e0df2c90
Signed-off-by: Yuval Caduri <cyuval@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/5865
Reviewed-by: Liron Himi <liron.himi@marvell.com>
Reviewed-by: Nadav Haklai <Nadav.Haklai@cavium.com>
Reviewed-by: Stefan Chulski <Stefan.Chulski@cavium.com>
Tested-by: sa_ip-sw-jenkins
[RH: Original patch taken from marvell 88F3720 board support SDK 10.0-PR2003]
Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 .../devicetree/bindings/marvell-uio-xor.txt          | 12 ++++++++++++
 1 file changed, 12 insertions(+)
 create mode 100644 Documentation/devicetree/bindings/marvell-uio-xor.txt

diff --git a/Documentation/devicetree/bindings/marvell-uio-xor.txt b/Documentation/devicetree/bindings/marvell-uio-xor.txt
new file mode 100644
index 000000000000..a41c3e859b96
--- /dev/null
+++ b/Documentation/devicetree/bindings/marvell-uio-xor.txt
@@ -0,0 +1,12 @@
+* Marvell UIO for XOR driver
+
+Required properties:
+
+- compatible: should be "marvell,uio-xor-v2".
+- xor_access: phandler to XOR driver
+
+Example:
+P110_LABEL(uio_xor1) {
+	compatible = "marvell,uio-xor-v2";
+	xor_access = <&CP110_LABEL(xor1)>;
+};
-- 
2.17.1

